forge/test/operators/pytorch/eltwise_unary/test_unary.py

```diff
@@ -160,6 +165,35 @@ class TestParamsData:
     test_plan_implemented: TestPlan = None
     test_plan_not_implemented: TestPlan = None
 
+    no_kwargs = [
+        None,
+    ]
+
+    kwargs_clamp = [
+        {"min": 0.0, "max": 0.5},
+        {"min": 0.5, "max": 0.0},
+        {"min": 0.2},
+        {"max": 0.2},
+    ]
+
+    kwargs_pow = [
+        {"exponent": 0.0},
+        {"exponent": 0.5},
+        {"exponent": 2.0},
+        {"exponent": 10.0},
+        {"exponent": -2.0},
+        {"exponent": -1.26},
+        {"exponent": 1.52},
```

Comment:
To limit total number of tests, full list of kwargs should be combined with subset of input shapes. We probably need introduction of an another TestCllection and separation of lists of kwargs.
Subset of input shapes for the value testing should be considered. Input shapes from quick collection is maybe not sufficient.
Otoh full list of kwargs should be tested with all data formats.

Reply:
Added required changes.